lib/blood_contracts/core/tuple.rb

```diff
@@ -56,6 +62,21 @@ def inherited(new_klass)
         new_klass.failure_klass ||= TupleContractFailure
         super
       end
+
+      private
+
+      # Handle arguments passed as hash with string or
+      # symbol keys
```

Comment:
Minor, but matter of taste.
Could you, please, add documentation lines for args and kwargs params? :)

I'd like to see `inch` satisfied.

Reply:
✅